src/compile/data/stack.ts

```diff
@@ -17,7 +17,7 @@ function getStackByFields(model: UnitModel): string[] {
 
     const scale = isScaleChannel(channel) ? model.getScaleComponent(channel) : undefined;
     const _field = field(fieldDef, {
-      binSuffix: scale && hasDiscreteDomain(scale.get('type')) ? 'range' : 'start'
+      binSuffix: 'start'
```

Comment:
It's not but I have no idea why we ever would use range instead of start. It's the same logic that I removed from the other function.